### ISSUE TITLE
Publishing data

### ISSUE THREAD

**Issue by martindurant** [intake/intake#288]
As inspired by https://datasette.readthedocs.io/en/stable/publish.html

To provide an endpoint on the server, which takes a catalogue URL, data source spec or perhaps even (small amount of) data, and adds  the results to the available data-sets

**Comment by martindurant** [comment]
The above could perhaps also be done via the GUI, if running as a service/standalone app.

An alternative view worth thinking about, is being able to automatically stand up VMs serving a particular catalogue as a way to "share". That might be pretty heavyweight  compared to sharing the cat itself.

**Comment by kunakl07** [comment]
Thanks @martindurant ,The site is very useful as I needed to deploying data and was searching for one,for my gsoc project.

**Comment by martindurant** [comment]
You mean datasette was useful, or Intake was useful @kunakl07  :)

**Comment by kunakl07** [comment]
@martindurant datasette!!
I am still facing issue with intake,I have also referred [this](https://github.com/intake/intake/pull/249),but,I am still facing import error for intake as its showing "ModuleNotFoundError: No module named 'intake'"

**Comment by martindurant** [comment]
@kunakl07 , I would be very glad to help you solve your import error. Can you specify, please, exactly the steps you took to install Intake, and the system that you are using?

**Comment by kunakl07** [comment]
@martindurant 
"conda install -c intake intake"
After this I installed toolz, cloudpickle, delay, pyyaml

**Comment by martindurant** [comment]
Intake already depends on those things (except I don't know "delay"), so you shouldn't have had to install anything else. It may be that something in your environment is preventing you getting the most recent Intake.
Since Intake is now available on conda-forge, the install recommendation has changed to

```
conda install -c conda-forge intake
```
Would you mind trying? Indeed, if it is something in your current environment, then perhaps it would be better to do this in a new environment, like
```
conda create -n test_intake -c conda-forge intake ipython jupyter intake-sql intake-parquet 
```

**Comment by martindurant** [comment]
Your terminal seems to be incorrectly include the final "." in the URL: if you click on the http link above, it does actually work. I'll update the message to put a space in, so this doesn't happen to anyone else.

In any case, you will need the "intake-xarray" package, also available on conda-forge.

**Comment by kunakl07** [comment]
Yes @martindurant thanks,you are absolutely correc!!,The link works prefectly fine,but I am unable to load any of them,even after installing intake-xarray package.I think that my terminal might be working incorrectly.
Will inform as soon as I am able to install plugins.
